common/src/limits.rs

```diff
@@ -8,7 +8,7 @@ use crate::{
 #[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
 pub struct Limits {
```

Comment:
Only the backends need to know about this right?

Reply:
proto uses claims, not limits, but the limits file is part of the claims feature